start.js

```diff
@@ -15,11 +15,18 @@ const addTransactionForDividend = dividend => {
 }
 
 const calculateDividendsForCompetitor = competitor => {
-    const { controlled, growth, history, ownedStocks, stockVolume, name } = competitor
+    const { controlled, growth, history, stockTransactions, stockVolume, name } = competitor
 
     if (controlled) {
         return 0
     }
+	
+	let ownedStocks = 0
+	
+	for (let i = 0; i < stockTransactions.length; i++) {
+		const transaction = stockTransactions[i]
+		ownedStocks += transaction.amount
+	}

```

Comment:
Maybe something like this?

```js
const amounts = stockTransactions.map(t => t.amount)
const ownedStocks = amounts.reduce((a,b) => a + b, 0)
```

Annoying that js doesn't have a nice way to sum, but our friends over at Hovgaard might have a library available to us that makes for an easy `sum` function that could be used instead. For example, lodash has `_.sum`

Reply:
Sure, I just wanted to make sure, that the mod works again, because it's one of the nicest and realistic mods out there.
There is another issue (estimated income) in the current version, but due the randomized downs/ups of the stock market, this will be another PR :)
I wish you a happy new year!

Reply:
I appreciate it! I wanted to make a great mod to accomplish this, but due to the frequent changes to the mod API it became more of a chore than I bargained for.

Enjoy your New Year!